src/components/ReactFlagsSelect/ReactFlagsSelect.test.tsx

```diff
@@ -81,6 +93,18 @@ describe("ReactFlagsSelect", () => {
         expect(screen.queryByText("United States")).not.toBeInTheDocument();
       });
 
+      it("does not render a secondary label of the selected country if showSecondarySelectedLabel is false", () => {
+        render(
+          <ReactFlagsSelect
+            {...defaultProps}
+            selected="US"
+            showSecondarySelectedLabel={false}
+            customLabels={{ US: { primary: "US", secondary: "+1" } }}
+          />
+        );
+        expect(screen.queryByText("United States")).not.toBeInTheDocument();
```

Comment:
This test does not check if the secondary label in the document.
It should also check if the primary label is in the document